src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+  {
+    Question: 
+      "In Python, what module supports regular expressions?",
+    Answer: "re",
+    Distractor1: "regex",
+    Distractor2: "pyregex",
+    Distractor3: "pythonre",
+    Explanation: 
+      "The Python \"re\" module provides regular expression support.",
```

Comment:
I don't think we need the `\\` here
```suggestion
      "The Python re module provides regular expression support.",
```